src/redux/app/reducer.ts

```diff
@@ -39,6 +42,7 @@ const initialState: AppState = {
     canvasToShowStatus: LoadingStatus.init,
     sidePanelMode: SidePanelMode.CLOSE,
     selectedStation: 'linestart',
+    selectedColine: 0,
```

Comment:
Should this be `undefined` by default? As the side panel has to handle the undefined case anyway.